### PR TITLE
Cleanup: Clang format, fix spelling, remove debug code, ...

### PR DESCRIPTION
**Pull request by TomFischer**
For details see commit messages.

### CODE REVIEW

File: Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp [hunk 1]
```diff
@@ -518,29 +519,6 @@ void NodeWiseMeshPartitioner::writeASCII(const std::string& file_name_base)
     writeNodesASCII(file_name_base);
 }
 
-void NodeWiseMeshPartitioner::resetGlobalNodeIndices()
-{
-    for (std::size_t i = 0; i < _mesh->_nodes.size(); i++)
-    {
-        _mesh->_nodes[i]->setID(_nodes_global_ids[i]);
```
Comment (wenqing) [inline Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp]: If you removed these two members, please move Node::setID() to the private section as it was originally there before.

File: Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp [hunk 2]
```diff
@@ -518,29 +519,6 @@ void NodeWiseMeshPartitioner::writeASCII(const std::string& file_name_base)
...
-    }
-    // sort
-    std::sort(_mesh->_nodes.begin(), _mesh->_nodes.end(),
-              [](const MeshLib::Node* a, const MeshLib::Node* b) {
-                  return a->getID() < b->getID();
-              });
-}
-
-void NodeWiseMeshPartitioner::writeGlobalMeshVTU(
```
Comment (wenqing) [inline Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp]: Sometimes it is useful to have a global mesh for checking the correction of the node indexing after partitioning. It is Ok to remove it, the global mesh will be generated in a post processing.

File: Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp (hunk 1 above)
Comment (TomFischer) [inline Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp]: Commit e38cb1b makes `Node::setID()` and all the `protected` attributes and method now `privat`.

File: Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp (hunk 2 above)
Comment (TomFischer) [inline Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp]: In the future we'll need a post processing tool that converts the binary format to the vtu or pvtu. I think such a tool would be the proper place for the removed code.

Comment (wenqing) [inline Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp]: Good.

**Comment by wenqing** [comment]
:fast_forward:

**Pull request merged**